Makefile

```diff
@@ -380,16 +385,16 @@ IMAGE_NAMES := grafana/loki grafana/promtail grafana/loki-canary
 # push(app, optional tag)
 # pushes the app, optionally tagging it differently before
 define push
-	$(SUDO) $(TAG_OCI)  $(IMAGE_PREFIX)/$(1):$(IMAGE_TAG) $(IMAGE_PREFIX)/$(1):$(if $(2),$(2),$(IMAGE_TAG))
-	$(SUDO) $(PUSH_OCI) $(IMAGE_PREFIX)/$(1):$(if $(2),$(2),$(IMAGE_TAG))
+	$(SUDO) $(TAG_OCI)  $(IMAGE_PREFIX)/$(1):$(IMAGE_TAG) $(IMAGE_PREFIX)/$(1):$(2)
+	$(SUDO) $(PUSH_OCI) $(IMAGE_PREFIX)/$(1):$(2)
 endef
 
 # push-image(app)
-# pushes the app, if branch==master also as :latest and :master
+# pushes the app, also as :latest and :master
 define push-image
-	$(call push,$(1))
-	$(if $(filter $(GIT_BRANCH),master), $(call push,$(1),master))
-	$(if $(filter $(GIT_BRANCH),master), $(call push,$(1),latest))
+	$(call push,$(1),$(IMAGE_TAG))
+	$(call push,$(1),master)
```

Comment:
I think this is going to publish `:master` whenever we create a new tag, but that's probably fine. WDYT?

Reply:
Yeah, @slim-bean and me were aware of that but thought it was fine